src/test/e2e/bundle_test.go

```diff
@@ -581,3 +581,23 @@ func TestListVariables(t *testing.T) {
 		require.Contains(t, cleaned, "prometheus:\n  variables: []\n")
 	})
 }
+
+func TestPublishTag(t *testing.T) {
```

Comment:
thoughts on adding this test case to the TestBundleWithLocalAndRemotePkgs test? This tests isnt very fast, just seeing if we can kill two birds with one test 🐦